server/etcdserver/api/v3discovery/discovery.go

```diff
@@ -55,22 +55,8 @@ var (
 )
 
 type DiscoveryConfig struct {
-	Token     string   `json:"discovery-token"`
-	Endpoints []string `json:"discovery-endpoints"`
-
-	DialTimeout      time.Duration `json:"discovery-dial-timeout"`
-	RequestTimeOut   time.Duration `json:"discovery-request-timeout"`
-	KeepAliveTime    time.Duration `json:"discovery-keepalive-time"`
-	KeepAliveTimeout time.Duration `json:"discovery-keepalive-timeout"`
-
-	InsecureTransport  bool   `json:"discovery-insecure-transport"`
-	InsecureSkipVerify bool   `json:"discovery-insecure-skip-tls-verify"`
-	CertFile           string `json:"discovery-cert"`
-	KeyFile            string `json:"discovery-key"`
-	TrustedCAFile      string `json:"discovery-cacert"`
-
-	User     string `json:"discovery-user"`
-	Password string `json:"discovery-password"`
+	clientv3.ConfigSpec `json:"client-spec"`
```

Comment:
nit: I would just call it `client` to not confuse whether its `spec` or `config`... but I don't have strong opinion here.

Reply:
Updated & thanks.